build.gradle

```diff
@@ -38,9 +38,9 @@ shadowJar {
     archiveClassifier = null
 }
 
-//checkstyle {
-//    toolVersion = '8.23'
-//}
+checkstyle {
+    toolVersion = '8.23'
+}
```

Comment:
Pull request is failing due to enabling of the gradle checkstyle. This means there are files that are failing the checkstyle, please amend thanks